src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenLocalGroupRpcRequest.java

```diff
@@ -21,21 +21,21 @@
 import java.io.IOException;
 import com.rapid7.client.dcerpc.io.PacketOutput;
 import com.rapid7.client.dcerpc.messages.RequestCall;
-import com.rapid7.client.dcerpc.mssamr.objects.DomainHandle;
+import com.rapid7.client.dcerpc.mssamr.objects.UserHandle;
 
 public class SamrOpenLocalGroupRpcRequest extends RequestCall<SamrOpenLocalGroupRpcResponse> {
```

Comment:
OP_NUM 27 should be SamrOpenAlias

https://msdn.microsoft.com/en-us/library/cc245751.aspx